host-bin/enter-chroot

```diff
@@ -338,6 +338,15 @@ tmpfsmount() {
     mount -i -t tmpfs -o "rw${2:+,}$2" tmpfs "$target"
 }
 
+# Determines currently-logged in user
+getactiveuser() {
+    local activeuser="Default"
+    if [ -x "/usr/bin/jq" ]; then
+        activeuser=$(jq -r .LastActiveUser /home/chronos/Local\ State)
```

Comment:
quote the path instead of using backslash escapes

Reply:
Done!